src/content/app/genome-browser/components/drawer/components/sequence-view/GeneSequenceView.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { noop } from 'lodash';
```

Comment:
`import noop from 'lodash/noop'`!

Otherwise it will import the whole lodash.

Reply:
Also, could you please make a similar change in this file: https://github.com/Ensembl/ensembl-client/blob/dev/src/content/app/genome-browser/components/zmenu/Zmenu.tsx#L19 Doesn't relate to this PR directly, but easy to do, and otherwise we might forget.

Reply:
Done